### PR TITLE
[elastic_items] Handle commas in filter-raw

### PR DESCRIPTION
**Pull request by valeriocos**
This code allows to handle commas defined in the values of the filter-raw param within the projects.json. To achive this, the split match is changed to include also the str `data.`.

Signed-off-by: Valerio Cosentino <valcos@bitergia.com>

### CODE REVIEW

File: grimoire_elk/elastic_items.py [hunk 1]
```diff
@@ -32,6 +32,9 @@
 HEADER_JSON = {"Content-Type": "application/json"}
 MAX_BULK_UPDATE_SIZE = 1000
 
+FILTER_DATA_ATTR = 'data.'
+FILTER_SEPARATOR = ',' + FILTER_DATA_ATTR
```
Comment (sduenas) [inline grimoire_elk/elastic_items.py]: I see a problem with is. People will forget about writing the next filter right after the comma. Some will write it after the comma and some other will leave one or several space characters.

**Review by sduenas — COMMENTED** [review]
Is there any way to improve what I wrote regarding the `,` and `data.`?

File: grimoire_elk/elastic_items.py (hunk 1 above)
Comment (valeriocos) [reply grimoire_elk/elastic_items.py]: It sounds reasonable, please @sduenas check the PR again. In a nutshell, a regular expression (,\s*data.) is used to split the filter. Tests have been added accordingly.

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**